deploy/compliance-operator/20-complianceOperator-Subscription.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "compliance-operator"
+  namespace: "openshift-compliance"
+spec:
+  channel: "4.6"
```

Comment:
Will this blow up 4.7 and 4.5 clusters?

Reply:
I dont think so. As I understand it, it is the operator's channel and _should_ work on +/-1 cluster version. I will double check with the dev team but since this will only be on our own stg clusters it shouldnt cause issues.